src/main/com/yetanalytics/datasim/random.clj

```diff
@@ -154,14 +154,18 @@
   (let [even-odds (/ 1 (count coll))]
     (apply max-key
            (fn [el]
-             (rand-gauss
-              rng
-              (+ even-odds (* sd (get weights el 0.0)))
-              sd))
+             (let [weight (get-in weights [el :weight] 0.0)]
+               (if (<= weight -1.0)
```

Comment:
could you explain this behavior change?

Reply:
yep! So i made it so that if it's -1 it doesn't happen. This sets the RESULT to -1 which is lower than the space for all the other values being compared, so it always loses. I suppose the nuke-from-orbit would be to filter coll before even starting this part though...